client-flink/common/src/main/java/org/apache/celeborn/plugin/flink/RemoteShuffleMaster.java

```diff
@@ -47,9 +55,7 @@ public class RemoteShuffleMaster implements ShuffleMaster<RemoteShuffleDescripto
   private ShuffleResourceTracker shuffleResourceTracker;
   private final ScheduledThreadPoolExecutor executor =
       new ScheduledThreadPoolExecutor(
-          1,
-          ThreadUtils.createFactoryWithDefaultExceptionHandler(
-              "remote-shuffle-master-executor", LOG));
+          1, ThreadUtils.namedThreadFactory("remote-shuffle-master-executor", LOG));
```

Comment:
is it necessary to propagate the `Logger` instance?

Reply:
@pan3793, IMO, it's necessary to propagate the Logger instance, which could directly know from which class the `Logger` instance is.